packages/fast-components-styles-msft/src/text-action/index.ts

```diff
@@ -70,14 +74,31 @@ const styles: ComponentStyles<TextActionClassNameContract, DesignSystem> = {
             borderColor: neutralOutlineActive,
         },
     },
+    textAction__filled: {
+        background: neutralFillRest,
+        border: format("{0} solid transparent", toPx<DesignSystem>(outlineWidth)),
+        "&:hover": {
+            background: neutralFillHover,
+            borderColor: "transparent",
+        },
+        "&:active": {
+            background: neutralFillActive,
+            borderColor: "transparent",
+        },
```

Comment:
same comment as above, you can consolidate the borderColor with `"&:hover, &:active"`

Reply:
Same as above.